valhalla/mjolnir/osmnode.h

```diff
@@ -43,6 +43,7 @@ struct OSMNode {
   // Lat,lng of the node
   float lng_;
   float lat_;
+  uint32_t spare2_;
```

Comment:
The only explanation is compiler ompimizes out `memset` for implicit padding. Files doesn't match without this fix

Reply:
this is the right thing to do anyway, without it on a different system you might get different behavior. basically this is a bug